src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -759,6 +778,9 @@ int dhcp_server_handle_message(sd_dhcp_server *server, DHCPMessage *message,
         existing_lease = hashmap_get(server->leases_by_client_id,
                                      &req->client_id);
 
+        static_lease = hashmap_get(server->static_leases_by_client_id,
+                                      &req->client_id);
```

Comment:
we usually indent arguments on continuation lines to the `(` of the function call, see immediately above.